slack/src/main/java/org/togglz/slack/sender/HttpPostRequest.java

```diff
@@ -0,0 +1,57 @@
+package org.togglz.slack.sender;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.ByteStreams;
+import com.google.common.io.CharStreams;
+import org.togglz.core.logging.Log;
+import org.togglz.core.logging.LogFactory;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+class HttpPostRequest {
+
+    private static final Log log = LogFactory.getLog(HttpPostRequest.class);
+
+    private final String requestUrl;
+
+    HttpPostRequest(String requestUrl) {
+        this.requestUrl = requestUrl;
+    }
+
+    String send(byte[] body) {
+        HttpURLConnection connection = null;
+        try {
+            URL url = new URL(this.requestUrl);
+            connection = (HttpURLConnection) url.openConnection();
```

Comment:
We should set connection and socket timeouts here. By default there are no timeouts, so in case of network problems this may hang forever, which is pretty long. ;-)